sot/opcode_translator/executor/variable_dispatch.py

```diff
@@ -122,6 +123,43 @@ def inner(*args, **kwargs):
         tracker=DummyTracker([]),
     ),
 )
+
+Dispatcher.register(
+    dict,
+    ("DictVariable",),
+    lambda var: var.copy(),
+)
+
+
+@Dispatcher.register_decorator(dict)
+def dispatch_dict(var: ListVariable | TupleVariable):
+    res_dict = {}
+    length_var = BuiltinVariable(len, var.graph, DanglingTracker())(var)
+    getitem = BuiltinVariable(operator.getitem, var.graph, DanglingTracker())
+    for index in range(length_var.get_py_value()):
+        index_value = getitem(var, index)
+        # check
+        assert isinstance(index_value, (ListVariable, TupleVariable))
+        assert len(index_value) == 2
+        # recombination
+        key = getitem(index_value, 0)
+        value = getitem(index_value, 1)
+        value.graph.add_global_guarded_variable(key)
+        res_dict.update({key.get_py_value(): value})
+    return DictVariable(res_dict, var.graph, DummyTracker([var]))
+
+
+@Dispatcher.register_decorator(dict.fromkeys)
+def dispatch_dict_fromkeys(seq: ListVariable | TupleVariable, default: VariableBase = None):  # type: ignore
+    if default is None:
+        default = ConstantVariable.wrap_literal(default, seq.graph)
```

Comment:
```suggestion
        default = ConstantVariable.wrap_literal(None, seq.graph)
```

这里的 None 表示函数默认值，default is None 表示 dispatch 没有传值，两者混在一起容易让人疑惑